ui/apps/platform/src/Containers/Vulnerabilities/utils/sortUtils.tsx

```diff
@@ -1,7 +1,15 @@
 import sortBy from 'lodash/sortBy';
-import { ensureExhaustive } from 'utils/type.utils';
+import { ensureExhaustive, isNonEmptyArray, NonEmptyArray } from 'utils/type.utils';
 import { SortAggregate, SortOption } from 'types/table';
-import { WorkloadEntityTab } from '../types';
+import { FieldOption } from 'hooks/useURLSort';
+import intersection from 'lodash/intersection';
```

Comment:
Small request to move `'lodash/intersection'` above ui imports in case of future change to lint style.